src/module-elasticsuite-catalog-optimizer/etc/logging.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<logging xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Logging:etc/logging.xsd">
+    <groups>
+        <group name="elasticsuite_optimizer">
+            <label translate="true">Manage Elasticsuite Search Optimizers</label>
+            <expected_models>
+                <expected_model class="Smile\ElasticsuiteCatalogOptimizer\Model\Optimizer" />
+            </expected_models>
+            <events>
+                <event controller_action="adminhtml_optimizer_save" action_alias="save" />
```

Comment:
Hello,
I think the prefix of the action should be the route id, not `adminhtml`
So `smile_elasticsuite_catalog_optimizer` i think
But I did not test this yet

Reply:
Like in the module @Magento_Invitation@ for example

Reply:
Can confirmed and just tested on a EE 2.4.3-p1.
The controller action syntax is "frontName_controller_action", the frontName being defined in the module's etc/adminhtml/routes.xml

For instance for Thesaurus, the frontName is "smile_elasticsuite_thesaurus" and not the native "adminhtml" :
```
    <router id="admin">
        <route id="smile_elasticsuite_thesaurus" frontName="smile_elasticsuite_thesaurus">
            <module name="Smile_ElasticsuiteThesaurus" before="Magento_Backend" />
        </route>
    </router>
```